cmd/node/factory/structs.go

```diff
@@ -2010,6 +2011,7 @@ func newMetaBlockProcessor(
 		RepairCallbackEnableEpoch:           generalConfig.GeneralSettings.RepairCallbackEnableEpoch,
 		ReturnDataToLastTransferEnableEpoch: generalConfig.GeneralSettings.ReturnDataToLastTransferEnableEpoch,
 		SenderInOutTransferEnableEpoch:      generalConfig.GeneralSettings.SenderInOutTransferEnableEpoch,
+		RepairScrNonceIncrementEnableEpoch:  generalConfig.GeneralSettings.RepairScrNonceIncrementEnableEpoch,
```

Comment:
Maybe rename to IncrementSCRNonceInMultiTransfer? Also this could have gone to development.

Reply:
done